cmd/crank/render/cmd.go

```diff
@@ -81,13 +91,13 @@ func (c *Cmd) Run() error { //nolint:gocyclo // Only a touch over.
 
 	s := json.NewSerializerWithOptions(json.DefaultMetaFactory, nil, nil, json.SerializerOptions{Yaml: true})
 
-	fmt.Println("---")
+	fmt.Fprintln(k.Stdout, "---")
```

Comment:
Perhaps https://pkg.go.dev/github.com/alecthomas/kong#Kong.Printf?

Reply:
Oh never mind. I see `k.Printf` includes the application name as a prefix, which we don't want.